File: src/pytorch/model_benchmarks_cpu.py
from torchvision.models import vgg16, vgg16_bn, vgg19, vgg19_bn, resnet18, resnet34, resnet50, resnet101, resnet152
from benchmarking_cpu import run_benchmarks

n_iters = 51
batch_size = 8

print("The models are being run on input size of {}".format((batch_size, 3, 224, 224)))
print("The time is the average over {} iterations".format(n_iters-1))
run_benchmarks([vgg16, vgg16_bn, vgg19, vgg19_bn, resnet18, resnet34, resnet50, resnet101, resnet152], batch_size, n_iters)

File: src/flux/model_benchmarks_cpu.jl
include("vggmodels.jl")
include("resnetmodels.jl")
include("benchmarking_cpu.jl")

using Flux

batch_size = 8

run_benchmarks([vgg16, vgg16_bn, vgg19, vgg19_bn, ResNet18, ResNet34, ResNet50, ResNet101, ResNet152], batch_size)
